packages/interface-default/src/animation-blur.module.css

```diff
@@ -0,0 +1,23 @@
+.enter {
+  backdrop-filter: blur(0px);
+}
+
+.enterActive {
+  backdrop-filter: blur(4px);
+  transition-property: backdrop-filter;
+  transition-timing-function: ease-in-out;
+}
+
+.enterDone {
+  backdrop-filter: blur(4px);
```

Comment:
@todo revert the animation logic:

css of the animated element should have visible status by default, the animation then adds the in/out transitions.

That way we do not need the enterDone class

Reply:
na that might make it more complicated or result in flickering elements. Lets keep it